docs/README.md

```diff
@@ -1,20 +1,27 @@
 ## Building Docs Locally
 ### First Time Setup:
+Create a new conda environment:
+
+```bash
+# We require python>=3.9 and cmake>=3.10
+conda create -n habitat python=3.9 cmake=3.14.0
+conda activate habitat
+```
+
 Install required packages:
 ```
-conda install -c conda-forge doxygen==1.8.16
-conda install jinja2 pygments docutils
+cd habitat-sim
+pip install -r requirements.txt --progress-bar off
+pip install imageio imageio-ffmpeg
+conda install -y -c conda-forge doxygen=1.9.5
+conda install -y  jinja2 pygments docutils
 ```
 
-To build the docs, you will also need to install the following
-packages on Linux :
+To build the docs, you will also need to install textlive on Fedora Linux :
 ```
-sudo apt install --allow-change-held-packages \
-                  texlive-base \
-                  texlive-latex-extra \
-                  texlive-fonts-extra \
-                  texlive-fonts-recommended
+sudo dnf install texlive-scheme-full
 ```
+
```

Comment:
Let's leave both options. The `apt install` option is for Debian systems like Ubuntu.

Reply:
I agree, let's keep both